third-party/linshare/src/test/java/org/apache/james/linshare/LinshareExtension.java

```diff
@@ -109,6 +131,32 @@ public LinshareConfiguration configurationWithJwtFor(LinshareFixture.Credential
             .downloadShare(document.asString(), filename);
     }
 
+    @Override
+    public void beforeAll(ExtensionContext extensionContext){
+        TechnicalAccountCreationRequest technicalAccount = new TechnicalAccountCreationRequest(TECHNICAL_ACCOUNT.getUsername(),
+            "technical@linshare.org",
+            "true",
+            TECHNICAL_ACCOUNT.getPassword(),
+            "DELEGATION");
+
+        createTechnicalAccount(technicalAccount);
```

Comment:
It's too soon to create a technical account ahead of every tests (which is necessary for the later task), having a method creating a technical account in the extension is enough for me